servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -71,6 +71,12 @@ quarkus.resteasy.gzip.enabled=true
 quarkus.http.auth.basic=false
 quarkus.oidc.enabled=false
 
+## quarkus kubernetes settings (https://quarkus.io/guides/deploying-to-kubernetes)
+quarkus.kubernetes.name=nessie-server
+quarkus.kubernetes.version=0.8.3
```

Comment:
Can you add some automation in `release-create.yml` GH workflow to update this?

Reply:
I removed this as this isn't really necessary to have in the config